### ISSUE TITLE
Figure out why Multispade is overfitting

### ISSUE THREAD

**Issue by andrewjong** [andrewjong/ShineOn-Virtual-Tryon#88]
# Train:
![image.png](https://images.zenhubusercontent.com/5d7fca0846611c000130d6b7/75f9e716-13ab-4829-a8d4-4332b9957f73)
# Validation:
![image.png](https://images.zenhubusercontent.com/5d7fca0846611c000130d6b7/73f37fc4-ac1f-4f90-9be7-d490b3e1ac8d)

Things to try
- Fix bug of person parts
- Dropout
- Reduce num layers
- Data augmentation
